src/doing/utils.py

```diff
@@ -223,16 +223,20 @@ def run_command(command: str, allow_verbose=True):
 
     encoding = guess_shell_encoding()
 
-    process = subprocess.run(
-        command,
-        stdin=subprocess.PIPE,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        universal_newlines=True,
-        encoding=encoding,
-        shell=True,
-        timeout=15,
-    )
+    try:
+        process = subprocess.run(
+            command,
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=True,
+            encoding=encoding,
+            shell=True,
+            timeout=15,
+        )
+    except Exception as e:
+        console.print(e)
```

Comment:
Before print the error, I'd also like to see the encoding used and the command trying to run. Something like:

```python
console.print(f"There was an error. Ran the following command with encoding '{encoding}':")
console.print(f"[bright_black]{command}[/bright_black]")
console.print(f"[dark_orange3]{e}[/dark_orange3]")
```

Reply:
Fixed by only catching timeout